composition/content_merge.go

```diff
@@ -178,7 +202,12 @@ func (cntx *ContentMerge) GetBodyFragmentByName(name string) (Fragment, bool) {
 
 func (cntx *ContentMerge) AddContent(c Content, priority int) {
 	cntx.addHead(c.Head())
-	cntx.addBodyAttributes(c.BodyAttributes())
+	content2, ok := c.(Content2)
```

Comment:
I would call it contentV2 to be clear that the naming choice is a result of a api change